website/content/docs/concepts/jobs.mdx

```diff
@@ -0,0 +1,79 @@
+---
+layout: docs
+page_title: Jobs
+description: |-
+  Learn about Nomad's Job feature, which is how you deploy your apps.
+---
+
+# Jobs
+
+This page explains the concept of a job, which is the Nomad construct for
+deploying your app and its configuration.
+
+
+## Background
+
+A job is a declarative specification of tasks that Nomad should run. Each job
+can have one or more task groups, which are collections of one or more tasks.
+
+Declare the the desired state of your job in a job specification that describes
+the type of job, the tasks and resources necessary for the job to run, and job
+information such as which clients it can run on. Refer to the [Job
+Specification](/nomad/docs/job-specification) for complete details and examples.
+
+When you submit your job specification, Nomad creates a job along with
+[allocations](/nomad/docs/glossary#allocation) for each group defined in the
+spec. Nomad makes sure the actual state matches your desired state.
+
+## Job types
+
+Nomad supports the following job types:
+
+- Batch
+
+  Batch jobs are for short lived jobs that run until they exit
+  successfully. Batch jobs are paramaterized or periodic.
```

Comment:
I don't know if it's true that batch jobs are parameterized or periodic, but it is true that there are Periodic and Parameterized job types that can have batch type child jobs.

(Batch type jobs can also be their own non-parameterized, non-periodic job type. Batch type is meant for running "scripts", or any other short-lived task)

I would consider adding Periodic and Parameterized jobs as types in addition to the ones here.

I would also consider grouping or tagging job types such that they are related across a couple important axes. For example: System and Sysbatch jobs are one-per-node, where others can have their placement configured; Batch and Sysbatch jobs are expected to be short-lived where Service and System jobs are expected to be relied about (like discoverable services)

Reply:
Also noting that there's some overlap here with https://developer.hashicorp.com/nomad/docs/schedulers

Reply:
I decided to remove this section. It's more involved that I first thought, with conflicting information across older docs and tutorials. I think this topic is better addressed when we port content from tutorials to docs.